Sources/ParseSwift/Objects/ParseSession.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ParseSession.swift
+//  ParseSwift
+//
+//  Created by Corey Baker on 1/17/21.
+//  Copyright © 2021 Parse Community. All rights reserved.
+//
+
+import Foundation
+
+/**
+ `ParseSession` is a local representation of a session.
+ This protocol conforms to `ParseObject` and retains the
+ same functionality.
+ */
+public protocol ParseSession: ParseObject {
+    associatedtype SessionUser: ParseUser
+
+    /**
+     The session token for this session.
+     */
+    var sessionToken: String { get }
+
+    /// The user the session is for.
+    var user: SessionUser { get }
+
+    /// Whether the session is restricted.
+    var restricted: Bool { get }
```

Comment:
I think this could be an issue, see https://github.com/parse-community/parse-server/issues/6612

Reply:
I see, I think making it "optional" should make it backwards/forwards compatible with Parse Server. What do you think?

Reply:
I can also remove it, developers just won't be able to see it in their Swift apps because it's never decoded

Reply:
The only potential problem I see with removing is it will show in their Parse Dashboard, but not on their swift apps...

Reply:
Making it optional seems like an acceptable option. I wonder whether the comment should be changes to say that the feature isn't implemented server side and/or link to the #6612